lib/matplotlib/axes/_axes.py

```diff
@@ -4384,7 +4387,14 @@ def _parse_scatter_color_args(c, edgecolors, kwargs, xsize,
             if facecolors is None:
                 facecolors = kwcolor
 
-        if edgecolors is None and not mpl.rcParams['_internal.classic_mode']:
+        edge_from_c = False
+        if edgecolors is None and c is not None:
+            edge_from_c = True
+
+        facecolors_none = isinstance(facecolors, str) and facecolors == 'none'
```

Comment:
```suggestion
        facecolors_none = cbook._str_lower_equal(facecolors, 'none')
```

Actually should probably be this for consistency.

Reply:
Updated